cache/s3proxy/s3proxy.go

```diff
@@ -49,6 +49,7 @@ func New(
 	// S3CloudStorageConfig struct fields:
 	Endpoint string,
 	Bucket string,
+	BucketLookupType minio.BucketLookupType,
```

Comment:
Could you add a command line flag for this too, along with the other `--s3.*` options?

Reply:
Added, PTAL.